ext/ftp/tests/ftp_ssl_connect_error.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test ftp_ssl_connect() function : error conditions
+--SKIPIF--
+<?php
+$ssl = 1;
+require 'skipif.inc';
+if (!function_exists("ftp_ssl_connect")) die("skip ftp_ssl is disabled");
+?>
+--FILE--
+<?php
+echo "*** Testing ftp_ssl_connect() function : error conditions ***\n";
+echo "\n-- Testing ftp_ssl_connect() function on failure --\n";
+var_dump(ftp_ssl_connect('127.0.0.1'));
+
+echo "\n-- Testing ftp_ssl_connect() function timeout warning for value 0 --\n";
+ftp_ssl_connect('127.0.0.1', 21, 0);
+
+?>
```

Comment:
will do. i'll cover invalid for each of the 3 arguments and also cover exceeds expected number of arguments

I don't possess enough understanding of the c code to figure out how to cover this portion:

    if (zend_parse_parameters(ZEND_NUM_ARGS(), "s|ll", &host, &host_len, &port, 
    &timeout_sec) == FAILURE) {
    return;
     }